hw01_hello_otus/go.mod

```
@@ -1,3 +1,5 @@
-module github.com/fixme_my_friend/hw01_hello_otus
+module hw01_hello_otus
```

```suggestion
module github.com/alexvancasper/otus-go-prof/hw01_hello_otus
```

Чтобы модуль был доступен для импорта, в названии следует использовать полный путь, включая репозиторий

---

Про модули, это же везде написано, а я пропустил. Учту на будущее
Спасибо, Артем!